Documentation/Helm-Charts/operator-chart.md

```diff
@@ -137,6 +137,7 @@ The following table lists the configurable parameters of the rook-operator chart
 | `csi.topology.enabled` | Enable topology based provisioning | `false` |
 | `currentNamespaceOnly` | Whether the operator should watch cluster CRD in its own namespace or not | `false` |
```

Comment:
please share the result of testing on a live cluster

Reply:
tested on minikube. updated label for running cluster: `osd`, `mon`, and `rgw` pods were updated accordingly.